components/server/src/initialization/database.py

```diff
@@ -32,6 +33,7 @@ def init_database() -> Database:  # pragma: no cover-behave
     add_last_flag_to_reports(database)
     rename_ready_user_story_points_metric(database)
     rename_teams_webhook_notification_destination(database)
+    rename_axe_selenium_python_to_axe_core(database)
```

Comment:
These rename functions are interesting. They need to be created everytime an item in the datamodel is renamed right?

how long do these functions need to stay in here? Until every QT instance anywhere is updated?

Maybe it might be worth it see if we think of a generic way to solve this problem? (although it's not immediately clear to me how we could achieve that)

Reply:
> These rename functions are interesting. They need to be created everytime an item in the datamodel is renamed right?

Yes, correct.

> how long do these functions need to stay in here? Until every QT instance anywhere is updated?

Yes. Because we follow semantic versioning, they can be removed after a major update.

> Maybe it might be worth it see if we think of a generic way to solve this problem? (although it's not immediately clear to me how we could achieve that)

It may be possible to specify renames in the data model and have the server apply the renames dynamically?